docs/styles/extra.css

```
@@ -0,0 +1,9 @@
+.md-source__fact--version {
+    display: none;
+  }
+  .md-source__fact--stars::before {
+    margin-left: 0;
```

guess it wasn't important enough to work
```suggestion
    margin-left: 0 !important;
```